packages/lib-classifier/dev/components/App/App.js

```diff
@@ -165,7 +183,13 @@ class App extends React.Component {
       >
         <Box as='main'>
           <Box as='header' pad='medium' justify='end' gap='medium' direction='row'>
-            <label htmlFor="workflows">Change workflow</label>
+            <label htmlFor="locale">Language</label>
+            <select id="locale" defaultValue={locale} onChange={this.selectLocale}>
+              <option value='en'>English</option>
+              <option value='es'>Español</option>
+              <option value='fr'>Français</option>
```

Comment:
Rather than hardcode options here, I suggest grabbing `project.available_languages` array and using it for the language dropdown in the dev classifier. The languageMenu.js file might need to be copied over from app-project to do so.

Reply:
Good idea!